static/js/map/map.js

```diff
@@ -371,7 +371,7 @@ function createStartLocationMarker() {
     const lng = useStoredPosition ? pos.lng : serverSettings.centerLng
 
     var marker = L.marker([lat, lng], { draggable: settings.isStartLocationMarkerMovable }).addTo(markersNoCluster)
-    marker.bindPopup(`<div><b>${i8ln('Start location')}</b></div>`, { autoPan: autoPanPopup() })
+    marker.bindPopup(function () { return `<div><b>${i8ln('Start location')}</b></div>` }, { autoPan: autoPanPopup() })
```

Comment:
No need to use a function here, just HTML is allowed: https://leafletjs.com/reference-1.7.1.html#layer-bindpopup

Reply:
It's needed since the locale isn't loaded yet here, so the translation isn't applied. By using a function, we can lazy load the popup content and get the trannslation.